.github/workflows/test_branches.yml

```diff
@@ -337,6 +337,9 @@ jobs:
         conda list --show-channel-urls
         which python
         python --version
+        # As of May 2025, python 3.13/conda doesn't default have setuptools
+        # Manually install to bypass this issue
+        conda install setuptools
```

Comment:
How does this interact with the Windows-specific setuptools install below?  Can we combine them into one?  Does setuptools need to be installed as a separate `conda install`, or can it be lumped into the rest of the dependencies (there was evidence in the past that it was *way* faster to install everything with a single `conda install` command where possible).

Reply:
The problem is that we can't run line 344 (`python setup.py dependencies`) without setuptools installed, so this step needs to happen before anything else is able to run.

Reply:
As for how it interacts with the Windows part - both lines are still necessary because we install whatever setuptools first so we can actually parse the dependencies, and then later we say, "Okay, if needed, please change to a different version."